README.md

```diff
@@ -13,6 +13,8 @@ impress.js may not help you if you have nothing interesting to say ;)
 
 HOW TO USE IT
 ---------------
+### Getting Started Guide
```

Comment:
This probably shouldn't be checked into this PR.

Reply:
Yeah right. Will remove this